### ISSUE TITLE
AttributeError: 'FFMPEG_AudioReader' object has no attribute 'proc'

### ISSUE THREAD

**Issue by thenewguy** [Zulko/moviepy#1487]
```
Exception ignored in: <bound method FFMPEG_AudioReader.__del__ of <moviepy.audio.io.readers.FFMPEG_AudioReader object at 0x7fc339261cf8>>
Traceback (most recent call last):
  File "/home/vagrant/.tox/py36/lib/python3.6/site-packages/moviepy/audio/io/readers.py", line 264, in __del__
    self.close()
  File "/home/vagrant/.tox/py36/lib/python3.6/site-packages/moviepy/audio/io/readers.py", line 254, in close
    if self.proc:
AttributeError: 'FFMPEG_AudioReader' object has no attribute 'proc'
```

Commit a3787719bed2ca4834ae6dc2f04567e4c229e81a works, 2482d139dd5514463bc1985bbcb5203250957603 fails. Latest master fails similarly as well.

I see this error littered through our test logs for a variety of file types of audio & video.  If you have trouble reproducing it then I will try to be more detailed.

Only difference with master is a slight change in line numbers:
```
Exception ignored in: <bound method FFMPEG_AudioReader.__del__ of <moviepy.audio.io.readers.FFMPEG_AudioReader object at 0x7fcbc5e229b0>>
Traceback (most recent call last):
  File "/home/vagrant/.tox/py36/lib/python3.6/site-packages/moviepy/audio/io/readers.py", line 266, in __del__
    self.close()
  File "/home/vagrant/.tox/py36/lib/python3.6/site-packages/moviepy/audio/io/readers.py", line 256, in close
    if self.proc:
AttributeError: 'FFMPEG_AudioReader' object has no attribute 'proc'
```

**Comment by mondeja** [comment]
Could you share one of those example files?

**Comment by thenewguy** [comment]
@mondeja I added a test at https://github.com/Zulko/moviepy/pull/1489

**Comment by mondeja** [comment]
Fixed in #1490. Could you give a try with current master and report here if now works as expected?

**Comment by thenewguy** [comment]
@mondeja Yes that seems to have fix the problem I was observing.  I will submit to CI and report back if the issue persists anywhere else.  Thanks!

**Comment by thenewguy** [comment]
@mondeja looks like there are a few related issues going on here.  will work up some more test cases when I have time to step though and understand what is going on.  Have some more like
```
 Exception ignored in: <bound method FFMPEG_AudioReader.__del__ of <moviepy.audio.io.readers.FFMPEG_AudioReader object at 0x7f2e1cd710b8>>
 Traceback (most recent call last):
   File "/home/vagrant/.tox/py3/lib/python3.6/site-packages/moviepy/audio/io/readers.py", line 266, in __del__
     self.close()
   File "/home/vagrant/.tox/py3/lib/python3.6/site-packages/moviepy/audio/io/readers.py", line 256, in close
     if self.proc:
 AttributeError: 'FFMPEG_AudioReader' object has no attribute 'proc'
```
on different files now

**Comment by mondeja** [comment]
That should be only part of the traceback. You could share the full one, maybe turning on verbosity in your CI?

**Comment by mondeja** [comment]
The problem still persists @thenewguy?

**Comment by thenewguy** [comment]
> The problem still persists @thenewguy?

Sorry I haven't had a chance to follow recent commits. Did you correct the problem? I will have to run the tests again. I will provide the failing test cases. It's been a really busy week

**Comment by bralbral** [comment]
I had the same problem when using a file from [](https://ottverse.com/ffmpeg-convert-avi-to-mp4-lossless/)



`    video = mp.VideoFileClip(input_filepath)
  File "<decorator-gen-123>", line 2, in __init__
  File "D:\PycharmProjects\moneymaking\venv\lib\site-packages\moviepy\decorators.py", line 95, in wrapper
    return func(*new_args, **new_kwargs)
  File "D:\PycharmProjects\moneymaking\venv\lib\site-packages\moviepy\video\io\VideoFileClip.py", line 141, in __init__
    self.audio = AudioFileClip(
  File "<decorator-gen-55>", line 2, in __init__
  File "D:\PycharmProjects\moneymaking\venv\lib\site-packages\moviepy\decorators.py", line 95, in wrapper
    return func(*new_args, **new_kwargs)
  File "D:\PycharmProjects\moneymaking\venv\lib\site-packages\moviepy\audio\io\AudioFileClip.py", line 62, in __init__
    self.reader = FFMPEG_AudioReader(
  File "D:\PycharmProjects\moneymaking\venv\lib\site-packages\moviepy\audio\io\readers.py", line 62, in __init__
    self.bitrate = infos["audio_bitrate"]
KeyError: 'audio_bitrate'
Exception ignored in: <function FFMPEG_AudioReader.__del__ at 0x000001FE7162A430>
Traceback (most recent call last):
  File "D:\PycharmProjects\moneymaking\venv\lib\site-packages\moviepy\audio\io\readers.py", line 266, in __del__
    self.close()
  File "D:\PycharmProjects\moneymaking\venv\lib\site-packages\moviepy\audio\io\readers.py", line 256, in close
    if self.proc:
AttributeError: 'FFMPEG_AudioReader' object has no attribute 'proc'

Process finished with exit code 1
`

**Comment by mondeja** [comment]
> I had the same problem when using a file from url

Just to be sure, the url which you are referring to is [this](https://file-examples-com.github.io/uploads/2018/04/file_example_AVI_1280_1_5MG.avi), right @bralbral?

**Comment by bralbral** [comment]
> > I had the same problem when using a file from url
> 
> Just to be sure, the url which you are referring to is [this](https://file-examples-com.github.io/uploads/2018/04/file_example_AVI_1280_1_5MG.avi), right @bralbral?
yes

**Comment by bralbral** [comment]
> > I had the same problem when using a file from url
> 
> Just to be sure, the url which you are referring to is [this](https://file-examples-com.github.io/uploads/2018/04/file_example_AVI_1280_1_5MG.avi), right @bralbral?

yes

**Comment by mondeja** [comment]
Thanks for the report @bralbral, it's fixed in #1506.
